.github/workflows/build.yml

```
@@ -0,0 +1,35 @@
+name: Build branch
+on:
+  schedule:
+    - cron: "0 5 * * 1" # every monday at 5 am
+  push:
+    branches:
+      - "*"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.11
```

Hello, using a newer version would be better :
```suggestion
     name: Setup java
        uses: actions/setup-java@v3
        with:
          distribution: "temurin"
          java-version: 11
          cache: "gradle"
```

---

And it allows you to delete the next 2 tasks.